worker/wallet.js

```diff
@@ -143,19 +139,6 @@ export async function checkInvoice ({ data: { hash, invoice }, boss, models, lnd
     if (dbInv.actionType) {
       return await paidActionPaid({ data: { invoiceId: dbInv.id, invoice: inv }, models, lnd, boss })
     }
-
-    // XXX we need to keep this to allow production to migrate to new paidAction flow
-    // once all non-paidAction receive invoices are migrated, we can remove this
-    const [[{ confirm_invoice: code }]] = await serialize([
```

Comment:
Prod database has been fulling migrated to all deposits being `RECEIVE` paid actions